main.js

```diff
@@ -0,0 +1,6 @@
+const { deduplicateLeads } = require("./src/functions/deduplicateLeads");
```

Comment:
Your program is lacking a clear entry point. There are a bunch of files on the top level directory, but it's not apparent where the primary logic actually gets executed, and I had to dig around to find the call hidden at the bottom of the `deduplicateLeads` function. You were on the right track by separating the functions into clear, independent pieces and exporting one from each file, so this takes it one step further: this new `main.js` folder sits at the top level and imports everything it needs from the contents of the newly-created `src` file (your functions and data) and then makes the call to kick off the program logic.